src/libs/import.c

```diff
@@ -1634,6 +1634,43 @@ static void _set_files_list(GtkWidget *rbox, dt_lib_module_t* self)
   gtk_box_pack_start(GTK_BOX(rbox), GTK_WIDGET(d->from.w), TRUE, TRUE, 0);
 }
 
+static void _browse_basedir_clicked(GtkWidget *widget, GtkEntry *basedir)
+{
+  GtkWidget *topwindow = gtk_widget_get_toplevel(widget);
+  if(!GTK_IS_WINDOW(topwindow))
+  {
+    topwindow = dt_ui_main_window(darktable.gui->ui);
+  }
+  GtkWidget *filechooser = gtk_file_chooser_dialog_new(
+      _("select directory"), GTK_WINDOW(topwindow), GTK_FILE_CHOOSER_ACTION_SELECT_FOLDER, _("_cancel"),
+      GTK_RESPONSE_CANCEL, _("_open"), GTK_RESPONSE_ACCEPT, (char *)NULL);
+#ifdef GDK_WINDOWING_QUARTZ
+  dt_osx_disallow_fullscreen(filechooser);
+#endif
+
+  gtk_file_chooser_set_select_multiple(GTK_FILE_CHOOSER(filechooser), FALSE);
+  gchar *old = g_strdup(gtk_entry_get_text(basedir));
+  char *c = g_strstr_len(old, -1, "$");
+  if(c) *c = '\0';
+  gtk_file_chooser_set_current_folder(GTK_FILE_CHOOSER(filechooser), old);
+  g_free(old);
+  if(gtk_dialog_run(GTK_DIALOG(filechooser)) == GTK_RESPONSE_ACCEPT)
+  {
+    gchar *dir = gtk_file_chooser_get_filename(GTK_FILE_CHOOSER(filechooser));
+
+    // dir can now contain '\': on Windows it's the path separator,
```

Comment:
As a comment, I don't know what is better: In lut3d.c I've ended up replacing '/' by '\\' (line 1102).

Reply:
Since i basically ripped of that code off of export module I feel no strong feeling about it. However if we were to do winpaths->unixpaths i'd go with replacing that handing everywhere in separate PR.